### ISSUE TITLE
Half signed in

### ISSUE THREAD

**Issue by MatSarr** [Lokavaluto/lokavaluto-mobile-application#7]
I signed in, then quit the app, and when coming back later, from the main page I've got the sign in button, but if a go to the menu I'm still connected (account name and email visible) but I can access only to the map.

**Comment by farfromrefug** [comment]
@MatSarr not normal what account are you using?

**Comment by stephansainleger** [comment]
I do not have this issue. @MatSarr so you still have it?

**Comment by MatSarr** [comment]
no, now I've got an error 404 - My website each time I open the app
